docs/setup/README.adoc

```diff
@@ -195,6 +195,26 @@ PRODUCT_BUNDLE_IDENTIFIER = YOUR_IOS_BUNDLE_IDENTIFIER;
 ....
 --
 
+== Enabling Push Notifications
+In order for the app to receive push notifications you need to establish a Firebase Cloud-Messaging Service instance and provide the credentials to said instance. 
+
+More information on FCM can be found using the following link:
+
+* https://firebase.google.com/docs/cloud-messaging[Google Documentation on FCM]
+* https://rnfirebase.io/[Using React Native with Firebase]
+
+To connect the app to your FCM instance update the following files: 
+
+* *`link:../../../../tree/main/ios/Compass/AppDelegate.m[ios/Compass/AppDelegate.m]`*
+** Comment in line 25: `[FIRApp configure];`
```

Comment:
Did you mean uncomment?

Reply:
Nope

Reply:
Per default, the line is

`// [FIRApp configure];`

and if i am not mistaken, for the initialization to work it needs to be changed to

`[FIRApp configure];`

thereby turning the 'comment' into 'code' (-> uncomment) or am i just not seeing something?

Reply:
Maybe it wasn't the best way to phrase it. It's meant to be understand like: 
-  **"comment in" that line** (with "comment in" being a composed verb)
- **"comment in" as opposite of "comment out"** ("comment out" meaning the turning of a loc into a comment)

`// [FIRApp configure];` 
is a **"commented out"** loc (because we turned it into a comment)

`[FIRApp configure];` 
is a **"commented in"** loc (as it was by default "commented out" and now it is not)

Definitely to much justification of a phrase when I could just have changed it 😅

Reply:
Done - updated the phrase.